respy/config.py

```diff
@@ -43,6 +43,8 @@
     "constant": "1",
     "exp_a_square": "exp_a ** 2 / 100",
     "exp_b_square": "exp_b ** 2 / 100",
+    "up_to_nine_years_edu": "exp_edu <= 9",
+    "at_least_ten_years_edu": "10 <= exp_edu",
```

Comment:
Actually, I like this one because then you never have to think about bigger and smaller again. It looks like everything is on the number line ([see this post](http://llewellynfalco.blogspot.com/2016/02/dont-use-greater-than-sign-in.html)). I have not settled on whether it is easier to read ``exp_edu <= 9`` or ``exp_edu < 10``. What is your opinion? BTW still I can see why you would like to put the variable name first, because it is the most important detail. Your call.

Reply:
I understand the reasoning but must say that I disagree. Especially for the case at hand, where we describe a partition. But I don't have a very good reason for this, so you can leave it as is. 

I prefer weak inequalities.